app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    operation, file_in, file_to_move = command.split()
```

Comment:
Need to validate that command have exactly the 3 parameters

Reply:
Not fixed need to validate it before operation, file_in, file_to_move = command.split()

Reply:
Pin a screenshot, if your code will work with following(must raise error with description):
cp file1.txt
cp file1.txt file2.txt file3.txt